backend/api/Areas/Research/Controllers/ResearchFileController.cs

```diff
@@ -58,6 +59,106 @@ public IActionResult GetResearchFile(long id)
             return new JsonResult(_mapper.Map<ResearchFileModel>(researchFile));
         }
 
+        /// <summary>
+        /// Gets the activities for specified research file.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("{researchFileId:long}/activities")]
+        [HasPermission(Permissions.ResearchFileView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(List<ActivityInstanceModel>), 200)]
+        [SwaggerOperation(Tags = new[] { "researchfile" })]
+        public IActionResult GetActivitiesForResearchFile(long researchFileId)
+        {
+            //TODO: Fetch actual activity from
+            //var researchFile = _pimsService.ActivityService.GetAllByResearchFileId(researchFileId);
+            List<ActivityInstanceModel> activities = new List<ActivityInstanceModel>();
+            activities.Add(new ActivityInstanceModel()
```

Comment:
if we know that is going to return an activity instance model this should probably be faked at the repository level

Reply:
We don't have proper columns and mapping in PimsActivityInstance right now to even mock. 
That's the reason data is faked for the model directly.

Reply:
I understand that the model is incomplete at the moment, but it could still be retrieved from the repository and add the missing fields here instead of not calling the service at all. That way the wiring from controller to service to repository is done leaving less work for the next story

Reply:
Seems fair, changed.